scripts/warroom/treasury_divest/convex.py

```diff
@@ -0,0 +1,28 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_convex()
+    vault.init_curve_v2()
+
+    badger_fraxbp = vault.contract(r.treasury_tokens.badgerFRAXBP_f_lp)
+    wcvx_badger_fraxbp = vault.contract(r.convex.frax.wcvx_badger_fraxbp)
+    badger = vault.contract(r.treasury_tokens.BADGER)
+    usdc = vault.contract(r.treasury_tokens.USDC)
+    frax = vault.contract(r.treasury_tokens.FRAX)
+    fxs = vault.contract(r.treasury_tokens.FXS)
+
+    vault.take_snapshot(tokens=[badger, usdc, frax, fxs])
```

Comment:
think we can also add crv and cvx for the rewards here

Reply:
not seeing any crv/cvx rewards in snapshot?

Reply:
oh maybe it only claims the fxs and not the crv cvx?